ghutil/ghutil.go

```diff
@@ -409,26 +414,23 @@ func (ghc *GitHubClient) ProcessPullRequest(ctx context.Context, orgName string,
 			} else {
 				logging.Infof("  No action needed: [%s] label already added", LabelClaYes)
 			}
+
+			addReview(pullRequestNonComplianceReason, ReviewApprove)
 		} else /* !pullRequestIsCompliant */ {
-			labelsUpdatedForNonCompliance := false
 			// if PR doesn't have [cla: no] label, add it.
 			if !hasLabelClaNo {
 				addLabel(LabelClaNo)
-				labelsUpdatedForNonCompliance = true
 			} else {
 				logging.Infof("  No action needed: [%s] label already added", LabelClaNo)
 			}
 			// if PR has [cla: yes] label, remove it.
 			if hasLabelClaYes {
 				removeLabel(LabelClaYes)
-				labelsUpdatedForNonCompliance = true
 			} else {
 				logging.Infof("  No action needed: [%s] label already missing", LabelClaYes)
 			}
 
-			if labelsUpdatedForNonCompliance {
-				addComment(pullRequestNonComplianceReason)
-			}
+			addReview(pullRequestNonComplianceReason, ReviewRequestChanges)
```

Comment:
Another thing I just realized: this is a big change of behavior for `crbot`, so it would be nice to control this with a command-line flag, and to be able to keep the old behavior (adding labels) as an option as well.

In fact, it would probably be safest to keep the old behavior as default, and make the new behavior something users can enable with a command-line flag, so that folks can try it out, without being suddenly surprised with a new behavior when they sync + rebuild their binaries.